web/public/js/controllers/user-profile-controller.js

```diff
@@ -576,7 +579,7 @@ function cdUserProfileCtrl($scope, $rootScope, $state, $window, auth, cdUsersSer
   function loggedInUserIsParent() {
     if(!loggedInUser.data) return false;
     return _.find(parentsForUser.data, function (parentForUser) {
```

Comment:
Probably drop using this resolve and just profile.data.parents just like is done for loggedInUserIsChild